### PR TITLE
 Add a design-less listing of fake requests

### PR DESCRIPTION
**Pull request by AntonRaichuk**

### CODE REVIEW

File: src/pages/orders/index.tsx [hunk 1]
```diff
@@ -18,10 +32,40 @@ export function Orders() {
   return (
     <React.Fragment>
       <Header hasHeadline hasLangSelector />
-      <h1>orders</h1>
-      <Tooltip title="Logout" arrow>
-        <OutputIcon onClick={logout} sx={{ width: 30, height: 30, marginLeft: "auto", cursor: "pointer" }} />
-      </Tooltip>
+      <Content>
+        <h1 className="title">Orders</h1>
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:36]: Should we use `react-i18next` here so it's easier to translate later?

File: src/pages/orders/index.tsx [hunk 2]
```diff
@@ -18,10 +32,40 @@ export function Orders() {
...
+        <Tooltip title="Logout" arrow>
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:37]: Should we use `react-i18next` here so it's easier to translate later?

Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:37]: Should we use `react-i18next` here so it's easier to translate later?

File: src/pages/orders/index.tsx [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
 import OutputIcon from "@mui/icons-material/Output";
 import { Tooltip } from "@mui/material";
+import { RequestStatus, fakeRequests } from "../../others/helpers/requests";
+import { useLocationsQuery, useSuppliesQuery } from "../../others/contexts/api";
+import styles from "./orders.module.css";
 
 export function Orders() {
   const { t } = useTranslation();
   const { logout } = useAuth();
+  const [visibleRequestId, setVisibleRequestId] = useState(0);
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:19]: Should the initial value be `undefined` rather than `0` here?

Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:19]: Should the default value be `undefined` rather than `0` here?

File: src/pages/orders/index.tsx [hunk 4]
```diff
@@ -18,10 +32,40 @@ export function Orders() {
...
+        <h1 className="title">Orders</h1>
+        <Tooltip title="Logout" arrow>
+          <OutputIcon onClick={logout} sx={{ width: 30, height: 30, marginLeft: "auto", cursor: "pointer" }} />
+        </Tooltip>
+          <Spacer size={24} />
+          <div className={styles.flex}>
+            {fakeRequests.map((request) => (
+              <React.Fragment key={request.id}>
+                <Card className={styles.card} onClick={() => setVisibleRequestId(request.id)}>
+                  <Text variant="bold">Request #{request.id}</Text>
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:45]: same here, or is this temporary and we are expecting the strings to change?

Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:45]: same here

File: src/pages/orders/index.tsx [hunk 5]
```diff
@@ -18,10 +32,40 @@ export function Orders() {
...
+                </Card>
+                { visibleRequestId === request.id && <Card>
+                  {cityLookup[request.city_id].name}<br/>
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:48]: We should be able to format this appropriately without the use of the `<br/>` tags. Not sure how important it is to address now, if this is just serving mostly as a placeholder for now.

Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:48]: We should be able to format this appropriately without the use of the `<br/>` tags. Not sure how important it is to address now, if this is just serving mostly as a placeholder for now.

File: src/pages/orders/index.tsx [hunk 6]
```diff
@@ -18,10 +32,40 @@ export function Orders() {
...
+      <Content>
+        <h1 className="title">Orders</h1>
+        <Tooltip title="Logout" arrow>
+          <OutputIcon onClick={logout} sx={{ width: 30, height: 30, marginLeft: "auto", cursor: "pointer" }} />
+        </Tooltip>
+          <Spacer size={24} />
+          <div className={styles.flex}>
+            {fakeRequests.map((request) => (
+              <React.Fragment key={request.id}>
+                <Card className={styles.card} onClick={() => setVisibleRequestId(request.id)}>
```
Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:44]: It's not clear that this component is clickable in the UI, but now sure how important it is to address if this is temporary.

Comment (alisonelizabeth) [inline src/pages/orders/index.tsx:44]: It's not clear that this component is clickable in the UI, but now sure how important it is to address if this is temporary.

**Review by alisonelizabeth — COMMENTED** [review]
Hey @AntonRaichuk! Left a few comments. I'm not quite sure how you'd like me to review this one. Can you help me better understand the goal? If we're mocking the data + creating a temporary UX, I'm not sure I see the benefits of merging it in as-is. Let me know what you think!

File: src/pages/orders/index.tsx (hunk 1 above)
Comment (AntonRaichuk) [reply src/pages/orders/index.tsx:36]: I'm wondering if it's actually going to be counter productive right now.
The captions could easily change (Requests/Orders/etc) in future, so I'd prefer to do a large translation pass closer to the end.

File: src/pages/orders/index.tsx (hunk 2 above)
Comment (AntonRaichuk) [reply src/pages/orders/index.tsx:37]: Same comment as above

File: src/pages/orders/index.tsx (hunk 4 above)
Comment (AntonRaichuk) [reply src/pages/orders/index.tsx:45]: Same comment as above

**Comment by AntonRaichuk** [comment]
@alisonelizabeth 

> Can you help me better understand the goal? If we're mocking the data + creating a temporary UX, I'm not sure I see the benefits of merging it in as-is. Let me know what you think!

I'm trying to implement this screen - https://www.figma.com/file/s0s9cVUjwBNLbhpDPTu2TY/Frente-Brazuca?node-id=22%3A5795
but:
 1) we do not have the API ready - that's why I added some mock data temporarily
 2) i'm a FE noob, so not sure how to transfer styles from figma to html/js/ts - so right now i'm doing a scaffolding for that page, and hopefully a real FE eng can come and just add proper styles to make it look like figma

Alternative to me doing this change is to do nothing, in which case there will be no progress.

File: src/pages/orders/index.tsx (hunk 3 above)
Comment (AntonRaichuk) [reply src/pages/orders/index.tsx]: Done

**Comment by alisonelizabeth** [comment]
> we do not have the API ready - that's why I added some mock data temporarily
> i'm a FE noob, so not sure how to transfer styles from figma to html/js/ts - so right now i'm doing a scaffolding for that page, and hopefully a real FE eng can come and just add proper styles to make it look like figma

👍 makes sense, thanks!  I can pick up fixing the styles once this is merged. My availability is limited to ~1-2 hrs in the evenings, but will try to make progress.

**Review by alisonelizabeth — APPROVED** [review]

**Pull request merged**